lib/hydra/works/models/concerns/contained_files.rb

```diff
@@ -0,0 +1,17 @@
+module Hydra::Works::ContainedFiles
+  extend ActiveSupport::Concern
+
+  included do
+    directly_contains :files, has_member_relation: ::RDF::URI("http://pcdm.org/hasFile"), class_name: "Hydra::Works::File"
+  end
+
+  def thumbnail
+    files = []
+    self.files.each do |file|
+      files << file unless file.metadata_node.query(predicate: RDF.type, object: ::RDF::URI("http://pcdm.org/#Thumbnail")).map(&:object).empty?
```

Comment:
@mjgiarlo see projecthydra-labs/hydra-pcdm#80. HP already has the RDFVocabularies class in it to which these terms would be added.